tests/functional/pages/surveys/repeating_household/repeating-household.page.js

```diff
@@ -0,0 +1,12 @@
+import QuestionPage from '../question.page'
+
+class RepeatingHouseholdPage extends QuestionPage {
+
+  isNameDisplayed(name) {
+    var title = browser.getText('h1[class="section__title saturn"]')
```

Comment:
Might be worth adding an identifiable name or id for titles which takes the id of the schema item